dbfit-java/build.gradle

```diff
@@ -294,6 +308,74 @@ project('netezza') {
         compile project(':dbfit-java:core')
         runtime 'org.netezza:nzjdbc3:7.2@jar'
     }
+
+    task setConnectionProperties {
+        doLast {
+            ext.propertiesFile = 'TestDbConnectionDbFitNetezza.properties'
+            ext.jdbcDriverClass = 'org.netezza.Driver'
+            ext.jdbcUrlPrefix = 'jdbc:netezza://'
+            ext.delimiterType = 'row'
+
+            def props = new Properties()
+            file(propertiesFile).withInputStream { props.load(it) }
+            ext.userName = props.getProperty('username')
+            ext.passWord = props.getProperty('password')
+            def propService = props.getProperty('service')
+            def propServiceParts = propService.tokenize(':')
+            def propServiceHost = propServiceParts[0]
+            def propServicePort = propServiceParts[1]
+            def propDatabase = props.getProperty('database')
+
+            ext.database = propDatabase.isEmpty() ? ('/' + propDatabase) : ''
+            ext.jdbcUrl = jdbcUrlPrefix + propServiceHost + (propServicePort.isEmpty() ? (':' + propServicePort) : '')
+        }
+    }
+
+    integrationTest.dependsOn setConnectionProperties
+
+    integrationTest.doFirst {
+        copy {
+            from setConnectionProperties.ext.propertiesFile + '.default'
+            into '.'
+            rename '(.*).default', '$1'
+        }
+
+        copy {
+            from setConnectionProperties.ext.propertiesFile + '.custom'
+            into '.'
+            rename '(.*).custom', '$1'
+        }
+
+        dbdeploy(setConnectionProperties.ext.jdbcDriverClass,
+            setConnectionProperties.ext.jdbcUrl + setConnectionProperties.ext.database,
+            setConnectionProperties.ext.userName,
+	    setConnectionProperties.ext.passWord,
+	    setConnectionProperties.ext.delimiterType)
+    }
+
+    task setUpTestDatabase(dependsOn: setConnectionProperties) {
+        doLast {
+            sql(setConnectionProperties.ext.jdbcDriverClass,
+                setConnectionProperties.ext.jdbcUrl + '/system',
+                setConnectionProperties.ext.userName,
+                setConnectionProperties.ext.passWord,
+                "${rootProject.projectDir}/test_vm/scripts/netezza/sql/create-databases-netezza.sql",
+                setConnectionProperties.ext.delimiterType, true
+            )
+        }
+    }
+
+    task tearDownTestDatabase(dependsOn: setConnectionProperties) {
+        doLast {
+            sql(setConnectionProperties.ext.jdbcDriverClass,
+                setConnectionProperties.ext.jdbcUrl + '/system',
+                setConnectionProperties.ext.userName,
+                setConnectionProperties.ext.passWord,
+                "${rootProject.projectDir}/test_vm/scripts/netezza/sql/drop-databases-netezza.sql",
+                setConnectionProperties.ext.delimiterType, true
+            )
```

Comment:
I think I got that from the existing `ant.sql` function declaration but, yes, I'll change the new code.